site/public/js/grade-inquiry.js

```diff
@@ -1,9 +1,15 @@
 /* global buildCourseUrl, WebSocketClient */
 /* exported loadDraft, initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
 
+function getDraftContentKeyPrefix() {
+    const gradeable_id = window.gradeable_id;
+    const course = window.location.pathname.split('/').slice(2, 4).join('-');
```

Comment:
Splitting the course name manually seems pretty sketchy.  Why not just use the full URL (or a hash of it) as part of the key?